### ISSUE TITLE
Violated assertion when reloading a problem with OsiGrbSolverInterface

### ISSUE THREAD

**Issue by FlorianPommerening** [coin-or/Osi#124]
The attached program [main.cpp](https://github.com/coin-or/Osi/files/3341809/main.cpp.txt) loads an LP into the solver twice. This works with CPLEX and SoPlex but not with Gurobi.

The assertion is
```OsiGrbSolverInterface.cpp:3986: void OsiGrbSolverInterface::convertToRangedRow(int, double, double): Assertion `colmap_O2G != NULL' failed.```

**Comment by svigerske** [comment]
I don't have a Gurobi license installed at the moment. Does this change help:
```diff
--- src/OsiGrb/OsiGrbSolverInterface.cpp	(revision 2269)
+++ src/OsiGrb/OsiGrbSolverInterface.cpp	(working copy)
@@ -3679,6 +3679,7 @@
   delete[] auxcolind;
   auxcolind = NULL;
   auxcolindspace = 0;
+  nauxcols = 0;
 }
 
 /// converts a normal row into a ranged row by adding an auxiliary variable
```
?

**Comment by FlorianPommerening** [comment]
Yes, that fixed it for me. I have the same caveat as in #123: I only tested the patch with an old version (0.107.9) but I see no reason why it wouln't work with other version as well. Thanks for fixing this. Any chance this could make it into a tagged release soon?

**Comment by svigerske** [comment]
Fixed in the svn repo some hours ago. May come to github some day. Fix is in release 0.108.5.
